pkg/vault/contracts/Vault.sol

```diff
@@ -1240,6 +1240,7 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
                 vaultUnderlyingDeltaHint = (amountInUnderlying.toInt256() + bufferUnderlyingImbalance).toUint256();
                 underlyingToken.forceApprove(address(wrappedToken), vaultUnderlyingDeltaHint);
                 vaultWrappedDeltaHint = wrappedToken.deposit(vaultUnderlyingDeltaHint, address(this));
+                amountOutWrapped -= 1;
```

Comment:
What if you do this at the beginning of the function? https://github.com/balancer/balancer-v3-monorepo/pull/1153/files#diff-f03125e354a92ba020d9db1ee551c1b9ba5034e99ae369a139e02e8679002709R1191 (same for exact out)